plugins/RocketsPlugin/jsonSerialization.h

```diff
@@ -217,12 +217,11 @@ inline void init(brayns::Renderer::PickResult* p, ObjectHandler* h)
 
 inline void init(brayns::Camera* c, ObjectHandler* h)
 {
-    h->add_property("look_at", Vector3dArray(c->_target), Flags::Optional);
+    h->add_property("quat", Vector4dArray(c->_quat), Flags::Optional);
```

Comment:
```suggestion
    h->add_property("quaternion", Vector4dArray(c->_quaternion), Flags::Optional);
```

Reply:
should be also 'orientation' as all the other instances

Reply:
Done